charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -631,6 +632,8 @@ def __init__(self, service, name=None, pg_num=None, replicas=None,
                          to this replicated pool.
         :type replicas: int
         :raises: KeyError
+        :param profile_name: Crush Profile to use (default: 'replicated_rule')
```

Comment:
With the default in the function above, the function signature documents this so isn't needed in a comment